tasks/facial_landmark_detection/README.md

```diff
@@ -0,0 +1,60 @@
+# Facial Landmarks
+
+This project demonstrates the use of Archai to find efficient neural architecture for facial landmarks localization. We follow the general methodology described in the papers [1] and [2], which use a synthetic face dataset and MobileNet V2 as the starting backbone.
```

Comment:
the best neural network architectures that optimize the tradeoff between validation accuracy and inference latency.

Reply:
How about "This project demonstrates how Archai can be used to find some really good neural network architectures that satisfy the specified objectives, namely, models that minimize the validation error and the inference latency".

Reply:
Do y ou like "superb"?